localization/english/culture/wc_cultural_innovations_l_english.yml

```diff
@@ -14,41 +14,50 @@
  ##############
  # Tribal Era #
  ##############
-
+ 
  innovation_amphibious:0 "Amphibious"
  innovation_amphibious_desc:0 "We surface on land only when we please, but being natives to the water we can travel the seas at ease."
 
+ innovation_hive_breeding:0 "Hive Breeding"
+ innovation_hive_breeding_desc:0 "Our massive hives allow us to breed countless workers and soldiers, always giving us the numerical advantage."
+ 
+ innovation_lorewalking:0 "Lorewalking"
+ innovation_lorewalking_desc:0 "Our ability to proficiently record history help us find ourselves through the lore of our ancestors."
+
  innovation_sea_faring:0 "Sea Faring"
```

Comment:
How about changing it to:
Living on the coasts gave us a natural advantage in not only taming, but conquering the seas and **trumping** those who dare to challenge us.